### DISCUSSION TITLE
Muncul Error : A value is trying to be set on a copy of a slice from a DataFrame.

### DISCUSSION THREAD

**Discussion by rahmalifa** [teamalgoritma/community#27] [Q&A] [answered]
Ketika menjalankan code, muncul error sebagai berikut. tetapi tetap berhasil keluar hasil outputnya
![image](https://user-images.githubusercontent.com/19927751/160114774-352de47a-acfc-4aa3-bb7e-69be31ba6255.png)

**Comment by hwulanayu** [comment, marked as answer]
Halo Ibu Rahma, SettingWithCopyWarning merupakan warning (bukan error) yang biasanya terjadi karena melakukan referencing pada suatu objek dan tidak menggunakan method copy(). Dengan melakukan referencing, objek asli dan objek yang direferensi akan mengarah/mempunyai ID yang sama, sehingga ketika salah satu objek diubah, objek lain akan ikut berubah nilainya. Mengenai referencing vs copying ada pada materi P4DA hari ke-empat.

**Reply by rahmalifa** [reply]
well noted, terimakasih jawabannya @hwulanayu :)
